### PR TITLE
[BKNDLSS-17927]: Change "secretKey" argument name to "apiKey"

### PR DESCRIPTION
**Pull request by JoeSilentJoe**
In every public API method in the code the "secretKey" argument renamed to "apiKey".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ Backendless SDK for Java and Android [![Build Status](https://travis-ci.org/Back
...
 
 1. **Create Developer Account.** An account is required in order to create and manage your Backendless backend. You can login to our console at: http://backendless.com/develop
-2. **Locate Application ID and Secret Key.** The console is where you can manage the applications, their configuration settings and data. Before you start using any of the APIs, make sure to select an application in the console and open the "Manage" section. The "App Settings" screen contains the application ID and secret API keys, which you will need to use in your code.
+2. **Locate Application ID and api Key.** The console is where you can manage the applications, their configuration settings and data. Before you start using any of the APIs, make sure to select an application in the console and open the "Manage" section. The "App Settings" screen contains the application ID and API keys, which you will need to use in your code.
```
Comment (markpiller) [inline README.md]: In the MD description it should be written as API

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ Backendless SDK for Java and Android [![Build Status](https://travis-ci.org/Back
...
 3. **Open Backendless Examples.** The SDK includes several examples demonstrating some of the Backendless functionality. The /samples folder contains an IDEA project file (AndroidSampleApps.ipr) combining all the samples. 
-4. **Copy/Paste Application ID and Secret Key.**  Each example must be configured with the application ID and secret key generated for your application. 
+4. **Copy/Paste Application ID and api Key.**  Each example must be configured with the application ID and api key generated for your application.
```
Comment (markpiller) [inline README.md]: here also API

File: samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Defaults.java [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+ *  ********************************************************************************************************************
+ */
+
+package com.backendless.examples.dataservice.tododemo;
+
+public class Defaults
+{
+	public static final String APPLICATION_ID = "";
+	  public static final String API_KEY = "";
+	  public static final String VERSION = "v1";
```
Comment (markpiller) [inline samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Defaults.java]: is this used anywhere? If not, it can be removed

Comment (JoeSilentJoe) [reply samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Defaults.java]: Used in samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Main.java
+36 Backendless.initApp( this, Defaults.APPLICATION_ID, Defaults.API_KEY, Defaults.VERSION );
Users mast enter their data in order to run the sample.
Similarly in another Defaults.
But can be change from
public static final String API_KEY = "";
to
public static final String API_KEY = "Your API KEY";

Comment (markpiller) [reply samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Defaults.java]: Does that example compile? We removed the version argument from the initApp call.

Comment (JoeSilentJoe) [reply samples/DataService/tododemo/src/com/backendless/examples/dataservice/tododemo/Defaults.java]: No, that dont compile. This sample contain another compile problem.
Fix this in https://github.com/Backendless/Android-SDK/pull/355

File: gradle.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
 signing.keyId=A6422129
 signing.password=123qwe
-signing.secretKeyRingFile=/Users/ksv/.gnupg/secring.gpg
+signing.apiKeyRingFile=/Users/ksv/.gnupg/secring.gpg
```
Comment (scadgek) [inline gradle.properties]: This is not related to Backendless: https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials
Please revert.

File: samples/MessagingService/pubsubdemo/src/com/backendless/examples/messagingservice/pubsubdemo/MainActivity.java [hunk 5]
```diff
@@ -35,9 +35,9 @@ public void onCreate( Bundle savedInstanceState )
     super.onCreate( savedInstanceState );
     setContentView( R.layout.main );
 
-    if( Defaults.APPLICATION_ID.equals( "" ) || Defaults.SECRET_KEY.equals( "" ) || Defaults.VERSION.equals( "" ) || Defaults.CHANNEL_NAME.equals( "" ) )
+    if( Defaults.APPLICATION_ID.equals( "" ) || Defaults.API_KEY.equals( "" ) || Defaults.VERSION.equals( "" ) || Defaults.CHANNEL_NAME.equals( "" ) )
     {
-      showAlert( this, "Missing application ID and secret key arguments. Login to Backendless Console, select your app and get the ID and key from the Manage > App Settings screen. Copy/paste the values into the Backendless.initApp call" );
+      showAlert( this, "Missing application ID and api key arguments. Login to Backendless Console, select your app and get the ID and key from the Manage > App Settings screen. Copy/paste the values into the Backendless.initApp call" );
```
Comment (scadgek) [inline samples/MessagingService/pubsubdemo/src/com/backendless/examples/messagingservice/pubsubdemo/MainActivity.java]: In comments and messages visible to end-users (like exception messages) it should be `API` (uppercase). Please review the changed files and replace such occurences.
